C-Plus-Plus/README.md

```diff
@@ -58,11 +58,18 @@
 - [Toggle Kth Bit From Right](cp/ToggleKthBitFromRight.cpp)
 - [Two Pointers Method](cp/two_pointers_method.cpp)
 - [Z Algorithm](cp/zalgorithm.cpp)
-<<<<<<< HEAD
-- [Peak element](cp/Peak_element.cpp)
 - [First occurence of 1](cp/first1_binsearch.cpp)
 - [Remove Minimum Parantheses to make Valid Parantheses](cp/removeMinParantheses.cpp)
->>>>>>> c8ecee37a2708f6788e13c1e0080f40635dd0702
+- [Moore's Voting Algorithm to find the majority element](cp/moore_voting.cpp)
```

Comment:
Do not make unnecessary changes in the README. Just add hyper link of your program under the correct heading.